cloud/cloud.go

```diff
@@ -496,4 +490,27 @@ func (c *Client) dumpRequest(req *http.Request) ([]byte, error) {
 	return httputil.DumpRequestOut(reqCopy, true)
 }
 
-const contentType = "application/json"
+func preparePayload(payload any) (io.Reader, string, error) {
```

Comment:
_nitpick_ that would help reading
```suggestion
func preparePayload(payload any) (body io.Reader, contentType string, err error) {
```